.gitignore

```diff
@@ -7,8 +7,10 @@
 ## Node.
 /node_modules
 /node/lib
-# Flatc generated files.
+# flatc generated files.
 /node/src/fbs
+# nvm version file.
+/.nvmrc
```

Comment:
Why was this necessary? You can ignore it locally, I don't see why it is necessary in the repo.

Reply:
I cannot ignore locally (meaning in my global .gitignore) since there are projects that **indeed** want this file into the repo. Just wondering what is so wrong with this addition in .gitignore taking into account that there are many other similar files in there such as ".idea" and so on.

Reply:
You can put whatever you want into `.git/info/exclude` of each repo, no need to put stuff not related to the project in `.gitignore`.

Reply:
Do you mean that such a "exclude" folder is per local repo (so you and me can have different things into it)?

BTW why did you never say anything about this? 😅

![image](https://github.com/versatica/mediasoup/assets/16191/f5af5803-9a60-4ebb-b15c-20d55e9aff1b)

Reply:
> Do you mean that such a "exclude" folder is per local repo (so you and me can have different things into it)?

Exactly, I usually put `.idea` in there when contributing to various projects, I don't expect all of them to use or care about JetBrains's IDEs.

> BTW why did you never say anything about this? 😅

Maybe I should have :man_shrugging:

Reply:
Ok I'll write PR later. I'll remove all those items but /.cache and /coverage which are autogenerated when running certain tasks. What about .tgz? Are we generating something.tgz and leaving it in root folder?

Reply:
I also add `/*.tgz` to `.gitignore` file, and also on `.npmignore`.

Reply:
We don't need .npmignore because we are explicit about npm package files in `package.json` (we fill the "files" field). Also, if `.npmignore` is not preset it reads `.gitignore`.

Reply:
No need for a PR. I've done basically what is told above: https://github.com/versatica/mediasoup/pull/1064/commits/96f0f45a3d3478e4f1d815f84b06da9b8b0cff75